src/libaktualizr/utilities/events.h

```diff
@@ -63,6 +63,14 @@ class PutManifestComplete : public BaseEvent {
   explicit PutManifestComplete();
 };
 
+/**
+ * No update is available for download from the server
```

Comment:
The tiniest of quibbles: I prefer my API docs to have sentence structure, i.e. end with a period.